cmd/cosign/cli/sign.go

```diff
@@ -69,13 +69,13 @@ func Sign() *ffcli.Command {
 		kmsVal      = flagset.String("kms", "", "sign via a private key stored in a KMS")
 		upload      = flagset.Bool("upload", true, "whether to upload the signature")
 		payloadPath = flagset.String("payload", "", "path to a payload file to use rather than generating one.")
-		forceTlog   = flagset.Bool("force-tlog", false, "whether to upload to the tlog even when the image is private")
+		force       = flagset.Bool("-f", false, "skip warnings and confirmations")
```

Comment:
Is requiring two dashes intentional here?

Reply:
d'oh. nope!